Applications: adding dotnix-follow-up

Ra33it0 opened the pull request:
#### Project Abstract

> Follow-up Application of Dotnix: Nix-based Polkadot Validators with an emphasis on security. 
>
> If your application is a follow-up to a previous grant, please mention which one in the first line of the abstract and include a link to previous pull requests if applicable.

#### Grant [level](https://github.com/w3f/Grants-Program#level_slider-levels)

- [ ] **Level 1**:  Up to $10,000, 2 approvals
- [x] **Level 2**:  Up to $30,000, 3 approvals
- [ ] **Level 3**:  Unlimited, 5 approvals (for >$100k: Web3 Foundation Council approval)

#### Application Checklist

- [x] The [application template](https://github.com/w3f/Grants-Program/blob/master/applications/application-template.md) has been copied and aptly renamed (`project_name.md`).
- [x] I have read the [application guidelines](https://github.com/w3f/Grants-Program/blob/master/docs/Support%20Docs/grant_guidelines_per_category.md).
- [x] Payment details have been provided (Polkadot AssetHub (USDC & DOT) address in the application and bank details via email, if applicable).
- [x] I understand that an agreed upon percentage of each milestone will be paid in vested DOT, to the Polkadot address listed in the application.
- [x] I am aware that, in order to receive a grant, I (and the entity I represent) have to successfully complete a KYC/KYB check.
- [x] The software delivered for this grant will be released under an open-source license specified in the application.
- [x] The initial PR contains only one commit (squash and force-push if needed).
- [x] The grant will only be announced once the first milestone [has been accepted](https://github.com/w3f/Grant-Milestone-Delivery#process) (see the [announcement guidelines](https://github.com/w3f/Grants-Program/blob/master/docs/Support%20Docs/announcement-guidelines.md)).

Ra33it0 committed:
Applications: adding dotnix-follow-up

semuelle commented on applications/dotnix-follow-up.md:
```suggestion
- **Total Costs:** 30,000 USD
```
?

semuelle commented:
@Lederstrumpf, @ajk-code, curious to hear your thoughts.

Ra33it0 committed:
dotnix-follow-up: add Total costs

Co-authored-by: Sebastian Müller <sebastian@web3.foundation>

semuelle commented on applications/dotnix-follow-up.md:
```diff
@@ -0,0 +1,236 @@
+- **Team Name:** Sporyon Technologies OÜ
+
+- **Payment Address-sporyon:** Fiat 25.03.2024, 15:18
+- **DOT-sporyon:** 1Z765yAFnVpzYT7Vk2wKaePoP1YGdYQ5yUw3nVEyu2DYju8
+- **DOT-tv:** 1Qtup4rW7TCCryMTH6UB7Fx2cN8ntbNyYVhbjRZiHP7skXQ
```
Please list one AssetHub address for USDC and one Polkadot address for DOT.

Ra33it0 replied:
Is it possible to receive the USD via bank transfer, as we did last time? 
I have listed two DOT addresses since tv and I would like to receive our DOT in two separate wallets.

semuelle replied:
Fiat payments are exclusive to [private grants](https://grants.web3.foundation/docs/Process/how-to-apply). If this is important, please apply through the form.

semuelle reviewed (changes requested):
Thanks again for the application and sorry for the long wait. I see that you integrated most of the feedback you received in the previous application, but could you summarize the changes from the old one to this one? 

Also, in the previous application you mentioned having plans for subsequent grants. We usually encourage teams to aim for independent funding, at least after the second grant, so could you comment on how you might fund further development (and maintenance) beyond this grant?

Ra33it0 commented:
To summarize the changes:
We have replaced the cloud deployment templates with a CI workflow using tools such as GitHub Actions, allowing users to create custom ISOs for deploying their hardened Polkadot validators. This approach is more architecture agnostic.

We also removed the VPN layer, CVE scanner, and monitoring and alerting features. 
Additionally, we have added Secure Boot implementation as well as SE-Linux. 
The Bash completion remains unchanged.

Regarding funding:
Since Dotnix is a Polkadot Validator implementation, we will try OpenGov if we cannot continue in the Grants program. In the case of maintenance, Dotnix's maintenance costs, as well as infrastructure, are paid by Sporyon Techologies, and we plan to continue doing so.

semuelle reviewed (approved):
Hey @Ra33it0, sorry for the long wait. Independent of whether you choose to apply in private or not, it seems that you have taken most of the suggestions from the previous applications to heart, so I am happy to support it.

Ra33it0 commented:
Hi @keeganquigley, we plan to release in the next couple of days. We have also implemented multi-validator support, which was not strictly part of this milestone but were a requested feature, as mentioned here [GitHub comment](https://github.com/w3f/Grant-Milestone-Delivery/pull/1278#issuecomment-3460577196). That took a little bit longer than expected.
